Add CPS ML datafeed troubleshooting docs for serverless (#7403)
elastic/docs-content: troubleshoot/_snippets/cps-ml-diagnostics-sources.md

> **Where to look**

Reviewer: ```suggestion
Use these sources to gather diagnostic information:
```

Author: Done in 2dc81b8b3.

> * **{{anomaly-job}} job messages in {{kib}}**: Open **Machine Learning → Anomaly Detection**, select the job, and review the **Job messages** tab for audit entries and warnings about linked projects, credentials, or scope changes. On the **Datafeed** tab, **View datafeed counts** opens the datafeed chart flyout for extraction timing.

Reviewer: we need a new var to use this phrasing - need to create in docset.yml

```suggestion
* **{{anomaly-job-cap}} job messages in {{kib}}**: Open **Machine Learning → Anomaly Detection**, select the job, and review the **Job messages** tab for audit entries and warnings about linked projects, credentials, or scope changes. On the **Datafeed** tab, **View datafeed counts** opens the datafeed chart flyout for extraction timing.
```

Author: Done in 2dc81b8b3 — added `anomaly-job-cap` to docset.yml and updated the snippet.

  The same entries are stored in `.ml-notifications-*`.

> * **`GET _ml/datafeeds/{datafeed_id}`**: Shows the effective `project_routing` value and, when an internal cloud credential exists, `authorization.cloud_api_key.id`.

Reviewer: ```suggestion
* **`GET _ml/datafeeds/{datafeed_id}`**: Shows the effective `project_routing` value and, when an internal cloud API key exists, `authorization.cloud_api_key.id`.
```

Author: Done in 2dc81b8b3.

* **`GET _ml/datafeeds/{datafeed_id}/_stats`**: While the {{dfeed}} runs, shows `remote_cluster_stats` with `total_clusters`, `available_clusters`, `skipped_clusters`, `availability_ratio`, `stabilized_cluster_aliases`, and `per_cluster_consecutive_skips`. The object is absent until the first search cycle establishes a baseline.

> * **`.ml-annotations-*`**: Scope-change annotations for the job. The annotation `event` field carries `search_scope_changed` (not the separate `type` field).

Reviewer 2: Should we tell them to use the alias, `.ml-annotations-read`?

* **`GET /_project/tags`**: Lists linked projects and their tags so you can compare them with a routing expression.
* **Elastic Cloud console**: Review linked projects in [Link and manage projects](/deploy-manage/cross-project-search-config/cps-config-link-and-manage.md).

> During active extraction problems, **Job messages** are authoritative. `remote_cluster_stats` from [get datafeed stats]({{es-apis}}operation/operation-ml-get-datafeed-stats) reflects the last **completed** cycle.

Reviewer: ```suggestion
If extraction failures are ongoing, check **Job messages** first. `remote_cluster_stats` from [get datafeed stats]({{es-apis}}operation/operation-ml-get-datafeed-stats) only updates after a cycle completes.
```

Author: Done in 2dc81b8b3 — shortened per your suggestion.
